### PR TITLE
#9 Maya v1.0.0 Sensors docs

### PR DESCRIPTION
**Pull request by lucasferryinbibo**
# Description of this issue
Sensors documented.

# Pages to review
sensors.md

Issue of this PR: https://github.com/Inbibo/adonisfx_docs/issues/9

### CODE REVIEW

**Comment by lucasferryinbibo** [comment]
Appoved on my behalf as I created the PR

File: docs/documentation/maya/sensors.md [hunk 1]
```diff
@@ -1,118 +1,182 @@
...
+AdnSensorPosition is the sensor for computing meaningful output values representing the velocity or acceleration of a transform node. The values of velocity an acceleration can then be remapped to produce desirable activation values to drive the simulation of the deformer. This sensor has to work in combination with an AdnLocatorPosition both for setup and visualization. And example use case for this sensor would be applying it to the wrist connection of an arm outputting velocities while swinging.
+
+### How to use
+
+An AdnSensorPosition will be in charge of feeding, after remapping, activation (or other) values into the deformer for driving the simulation and the AdnLocatorPosition for visualization purposes. The value of the sensor can be used to drive the activation of a muscle simulating contraction to increase its stiffness. Find more information for connecting the sensor to the deformer in sections below.
+
+<figure markdown>
+  ![AdnSensorPosition velocity display on AdnLocatorPosition within a scene](images/adn_sensor_position.png){width=60%}
+  <figcaption>Figure 1: AdnSensorPosition used in a human model.</figcaption>
```
Comment (josevalverdeinbibo) [inline docs/documentation/maya/sensors.md:15]: Add bold style to all "Figure x":
```suggestion
  <figcaption><b>Figure 1</b>: AdnSensorPosition used in a human model.</figcaption>
```

**Review by josevalverdeinbibo — APPROVED** [review]
Great job! Just one comment to check.

File: docs/documentation/maya/sensors.md (hunk 1 above)
Comment (josevalverdeinbibo) [reply docs/documentation/maya/sensors.md]: Done.

**Review by carlosmonteagudoinbibo — APPROVED** [review]

**Pull request merged**